apps/src/templates/instructions/codeReview/Comment.jsx

```diff
@@ -43,6 +44,9 @@ export default class Comment extends Component {
     );
   };
 
+  renderFormattedTimestamp = timestampString =>
```

Comment:
This is the only real logic change here, the changes below just reorganize the HTML more sensibly and stop using `float` to right align stuff (eg, ellipsis/comment options is now the last element listed in the HTML, which makes more sense structurally).